drivers/kscan/Kconfig.ft5336

```diff
@@ -11,3 +11,13 @@ config KSCAN_FT5336_PERIOD
 	int "Sample period (ms)"
 	default 10
 	depends on KSCAN_FT5336
+
+config KSCAN_FT5336_POLLING
```

Comment:
You don't need separate configs for polling and interrupt. You can remove `KSCAN_FT5336_POLLING` and use `#ifndef CONFIG_KSCAN_FT5336_INTERRUPT` in the driver instead

Reply:
Yeah, that makes sense, at first I was thinking someone might want to set both, but now I think about it, why would they :)